server.go

```diff
@@ -66,6 +66,13 @@ func (svr *Server) getHandle(handle string) (*os.File, bool) {
 	return f, ok
 }
 
+func (svr *Server) resolvePath(path string) string {
+	if filepath.IsAbs(path) {
```

Comment:
I don't think this is right, I think 

return filepath.Rel(srv.workingDir, path) is all that is required.